packages/@headlessui-react/src/components/combobox/combobox.tsx

```diff
@@ -1675,6 +1683,7 @@ function OptionsFn<TTag extends ElementType = typeof DEFAULT_OPTIONS_TAG>(
       '--button-width': useElementSize(data.buttonRef, true).width,
     } as CSSProperties,
     onWheel: handleWheel,
+    onScroll: handleScroll,
```

Comment:
Chose to use an `onScroll` event as none of the other events would fire if the first thing we move our mouse into is the native scrollbar.